### PR TITLE
[13.0][FIX] contract: portal showing "back to edit mode" for everyone

### PR DESCRIPTION
**Pull request by victoralmau**
[FIX] Portal showing "back to edit mode" for everyone
Fixes https://github.com/OCA/contract/issues/610

Please @joao-p-marques and @pedrobaeza review it.

@Tecnativa TT27781

### CODE REVIEW

**Comment by pedrobaeza** [comment]
About this, the form view shouldn't be the same if you return to a customer contract or supplier one. Can you check the method that return one or another depending on this?

File: contract/models/contract.py [hunk 1]
```diff
@@ -112,6 +112,20 @@ class ContractContract(models.Model):
         inverse_name="contract_id",
         string="Modifications",
     )
+    backend_view_id = fields.Integer(
+        store=False, compute="_compute_backend_view_id", string="Backend view"
+    )
+
+    def _compute_backend_view_id(self):
```
Comment (pedrobaeza) [inline contract/models/contract.py]: This is not the idea I had in mind. There's a standard method that returns the view to obtain. Can't this be used?

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Fast-tracking for fixing the problem:

/ocabot merge minor

is this happening in v12 as well?

**Comment by victoralmau** [comment]
Yes, in 12.0 it's equal, i will create PR to change it.

**Pull request merged**